compiler/natives/src/crypto/sha256/sha256.go

```diff
@@ -26,9 +26,25 @@ func webCryptoSum256(data []byte) (res [Size]byte, err error) {
 }
 
 func Sum256(data []byte) (res [Size]byte) {
-	if len(data) >= 2048 {
+	if len(data) >= 2600 {
 		// On big enough buffers, try to call the webcrypto implementation, it's much faster
-		// than the go implementation (typically 25x for a 10 MB buffer)
+		// than the go implementation. For small buffers though, webcrypto is much slower under Chrome,
+		// so we stay with the go implementation:
+		// +--------+-----------------------------------------------+
+		// |        |                Execution time                 |
+		// | Buffer +-----------------------+-----------------------+
+		// |        |       Chrome 55       |       Firefox 50      |
+		// | Length |-----------+-----------+-----------+-----------+
+		// |        |    Go     | webcrypto |    Go     | webcrypto |
+		// +--------+-----------+-----------+-----------+-----------+
+		// | 1B     | 83ns*     | 2.004ms   | 116ns*    | 149ns     |
+		// | 1kB    | 818ns*    | 2.003ms   | 334ns     | 152ns*    |
+		// | 2.5kB  | 2.072ms*  | 2.011ms   | 672ns     | 159ns*    |
+		// | 2.6kB  | 2.175ms   | 2.020ms*  | 689ns     | 159ns*    |
+		// | 2.7kB  | 2.400ms   | 2.022ms*  | 710ns     | 160ns*    |
+		// | 100kB  | 774ms     | 24ms*     | 215ms     | 16ms*     |
+		// | 10MB   | 1.632s    | 41ms*     | 1.76s     | 74ms*     |
+		// +--------+-----------+-----------+-----------+-----------+
```

Comment:
Also ran the 'stop benchmark after result, before sending result in channel' test and am seeing webcrypto faster until 64 bytes and less

len: 1 
go           20000	     58100 ns/op	   0.02 MB/s 
webcrypto     1000	   1267000 ns/op	   0.00 MB/s 
webcrypto No Chan       133800 ns/op
 
len: 64 
go           10000	    103200 ns/op	   0.62 MB/s 
webcrypto     1000	   1255000 ns/op	   0.05 MB/s 
webcrypto No Chan       134000 ns/op

len: 128
go           10000	    142300 ns/op	   0.90 MB/s 
webcrypto     1000	   1248000 ns/op	   0.10 MB/s 
webcrypto No Chan       134800 ns/op

len: 256 
go            5000	    221800 ns/op	   1.15 MB/s 
webcrypto     1000	   1258000 ns/op	   0.20 MB/s 
webcrypto No Chan       137200 ns/op

len: 512 
go            3000	    402000 ns/op	   1.27 MB/s 
webcrypto     1000	   1255000 ns/op	   0.41 MB/s 
 
len: 1024 
go            2000	    705000 ns/op	   1.45 MB/s 
webcrypto     1000	   1256000 ns/op	   0.82 MB/s 
 
len: 1200 
go            2000	    824000 ns/op	   1.46 MB/s 
webcrypto     1000	   1246000 ns/op	   0.96 MB/s 
 
len: 1500 
go            2000	   1012500 ns/op	   1.48 MB/s 
webcrypto     1000	   1254000 ns/op	   1.20 MB/s